modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java

```diff
@@ -1998,10 +2000,15 @@ private CommerceOrderItem _updateCommerceOrderItem(
 
 		CommerceOrder commerceOrder = commerceOrderItem.getCommerceOrder();
 
-		validate(
-			serviceContext.getLocale(), commerceOrder,
-			commerceOrderItem.getCPDefinition(),
-			commerceOrderItem.fetchCPInstance(), quantity);
+		boolean validateOrder = (Boolean)ParamUtil.getBoolean(
```

Comment:
I don't like retrieving the param here from the service context, we should avoid using the service context at all when we can. Add a parameter to the method and make it clear that we are passing the param from the form in the page. Thanks

Reply:
@luca-pellizzon Should I do method overloading to avoid breaking existing code?

Reply:
No go ahead and change directly the methods.